lib/build/loader.js

```diff
@@ -0,0 +1,82 @@
+"use strict"
+
+const path = require('path');
+const LoaderPlugin = require('./loader-plugin').LoaderPlugin;
+
+exports.generateLoaderConfig = function generateLoaderConfig(platform, bundler) {
```

Comment:
To clarify, we'd have something like `createLoaderCode` to generate the "code" that configures the loader (i.e. the whole code including `requirejs.config(theConfigObject)`), and then have `createLoaderConfig` be what returns just the configuration object for the loader. Sound good?